src/react/types.ts

```diff
@@ -0,0 +1,85 @@
+import type {
+  DocFragment,
+  DocResponse,
+  DocSet,
+  DocTypes,
+  DocWithId,
+  IndexKeyType,
+  IndexRow,
+  MapFn,
+  QueryOpts,
+  Database,
+  AllDocsQueryOpts,
+  ChangesOptions,
+  ClockHead,
+} from "@fireproof/core";
+
+export interface LiveQueryResult<T extends DocTypes, K extends IndexKeyType, R extends DocFragment = T> {
+  readonly docs: DocWithId<T>[];
+  readonly rows: IndexRow<K, T, R>[];
+  /** @internal */
```

Comment:
if internal why not private? 
or have a InternalLiveQueryResult and a External ?

Reply:
I changed them to be removed from the inferface. The purpose is for people who accidentally write `const docs` isntead of `const { docs }` it works as expected, but we dont want to document that in any way